x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/date_range_picker.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useDispatch } from 'react-redux';
+import React, { memo, useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import moment, { Moment } from 'moment';
+import { EuiFlexGroup, EuiFlexItem, EuiDatePicker, EuiDatePickerRange } from '@elastic/eui';
+
+import * as i18 from '../../translations';
```

Comment:
Are you using all the translations there? Otherwise you should import just the needed ones

Reply:
it's just the two translations, but they are deep nested within the export object.